mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/GroupRegistration.java

```diff
@@ -106,8 +106,8 @@ GroupRegistration start() {
         receiverSubscriber = Optional
             .of(createGroupWorkQueue()
                 .then(retryHandler.createRetryExchange(queueName))
-                .thenEmpty(this.consumeWorkQueue())
-                .subscribe());
+                .then(Mono.fromCallable(() -> this.consumeWorkQueue()))
```

Comment:
@mbaechler is it good for you?

Reply:
see https://github.com/linagora/james-project/pull/2173#discussion_r298920500